code/modules/admin/secrets.dm

```diff
@@ -108,6 +109,11 @@
 				dat += "No-one has done anything this round!"
 			usr << browse(dat, "window=admin_log")
 
+		// DS13 - Mentor system -- access mentor log
+		if("mentor_log") 
+			MentorLogSecret() 
+		// DS13 - Mentor system end
```

Comment:
DeepSpace13 - Mentor system (for consistency)

Reply:
Corrected